starters/solidjs-tanstack-tailwind/LICENSE

```diff
@@ -0,0 +1,7 @@
+Copyright 2022 This Dot 
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
```

Comment:
typo
limiting not limitation
and if we gonna use limitation it will be

_including without the limitation of the rights to use_

Reply:
You'll need to take that up with MIT. I didn't write the license, it's a copy of the text as provided.

Reply:
Oh, no problems then. I thought you did. Then we are good!

Thanks